site/operational-guide/uniform-dynamic-cluster.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Uniform Dynamic Clusters
+menu_order: 50
+---
+
+A uniform dynamic cluster has the following characteristics:
+
+* Each peer has identical configuration
+* Once an initial cluster has been bootstrapped, arbitrary numbers of
+  new peers can be added in parallel without coordination
+* Initial cluster peers can make progress from the outset even if
+  bootstrapping occurs under conditions of partition
```

Comment:
I'd add full stops after each item.

Unlike the "Manual/Incremental Deployment" section, this doesn't say what happens on reboot.